i18n/en.json

```diff
@@ -5702,6 +5710,14 @@
     "id": "model.websocket_client.connect_fail.app_error",
     "translation": "Unable to connect to the WebSocket server."
   },
+  {
+    "id": "mysqlsearchengine.purge-indexes.app-error",
```

Comment:
needed?

Reply:
Actually is not related at all, is related to local only files.